administrator/components/com_templates/src/View/Template/HtmlView.php

```diff
@@ -291,7 +292,28 @@ protected function addToolbar()
         }
 
         if (count($this->updatedList) !== 0 && $this->pluginState) {
-            ToolbarHelper::custom('template.deleteOverrideHistory', 'times', '', 'COM_TEMPLATES_BUTTON_DELETE_LIST_ENTRY', true, 'updateForm');
+            $dropdown = $bar->dropdownButton('override-group')
+            ->text('COM_TEMPLATES_BUTTON_CHECK')
+            ->toggleSplit(false)
+            ->icon('icon-ellipsis-h')
+            ->buttonClass('btn btn-action')
+            ->form('updateForm')
+            ->listCheck(true);
```

Comment:
The indentation should be increased here?
```suggestion
            $dropdown = $bar->dropdownButton('override-group')
                ->text('COM_TEMPLATES_BUTTON_CHECK')
                ->toggleSplit(false)
                ->icon('icon-ellipsis-h')
                ->buttonClass('btn btn-action')
                ->form('updateForm')
                ->listCheck(true);
```

Reply:
I think the calls should be indented like the rest as seen below.

Reply:
have merged the change but again the automated codestyle checker should have picked this up.